core/auth/fake/domain/identity.go

```diff
@@ -0,0 +1,34 @@
+package domain
+
+import (
+	"encoding/gob"
+
+	"flamingo.me/flamingo/v3/core/auth/mock"
+)
+
+type (
+	// Identity mocks auth.Identity
+	Identity struct {
+		mock.Identity
+		mock.OIDCIdentity
```

Comment:
`mock.OIDCIdentity` already embeds `mock.Identity`

Reply:
Also this does not properly fulfill the OIDCIdentity! It does have getters, but they will most likely panic or show nil results as there are no tokens available.
Also there is no support to specify claims. This will cause more trouble than added value im afraid.
See https://github.com/i-love-flamingo/flamingo/blob/master/core/auth/mock/identifier.go for how the identity behaves.